src/integrations/third-party/elementor.php

```diff
@@ -265,7 +265,8 @@ public function save_postdata() {
 			WPSEO_Meta::get_meta_field_defs( 'general', $post->post_type ),
 			WPSEO_Meta::get_meta_field_defs( 'advanced', $post->post_type ),
 			$social_fields,
-			WPSEO_Meta::get_meta_field_defs( 'schema', $post->post_type )
+			WPSEO_Meta::get_meta_field_defs( 'schema', $post->post_type ),
+			WPSEO_Meta::get_meta_field_defs( 'video', $post->post_type )
```

Comment:
I'm not sure I agree with adding any reference to video field defs in free. Am I missing something?